messages.md

```diff
@@ -95,6 +99,9 @@ These JSON hashes included in the table above are broken out into their own tabl
 |statistics.unsubscribed	|integer    |A read-only computed property representing the total number of messages where the person receiving the message unsubscribed.
 |statistics.bounced	|integer    |A read-only computed property representing the total number of messages that bounced or were otherwise undelivered.
 |statistics.spam_reports	|integer    |A read-only computed property representing the total number of messages marked as spam by people receiving them.
+|statistics.delivered       |integer    |A read-only computed property representing the SMS messages that were delivered to the recipients. (Applicable to SMS messages)

```

Comment:
Right Jason, have updated 'delivered' statistics for both Email and SMS messages.